near-plugins-derive/src/access_control_role.rs

```diff
@@ -46,6 +46,13 @@ use quote::quote;
 use std::convert::TryFrom;
 use syn::{parse_macro_input, ItemEnum};
 
+/// Roles as are represented by enum variants which are, in turn, represented by
+/// `u128` bitflags. Each variant requires two flags, one for the role itself
+/// and one for the corresponding admin permission. This would allow for 64
+/// roles. However, one flag is reserved for `__SUPER_ADMIN`, leaving 127
+/// bits that can fit 63 roles.
+pub const MAX_ROLE_VARIANTS: u8 = 63;
```

Comment:
Making it `pub` to have it surface in docs, since the assert message below mentions it.